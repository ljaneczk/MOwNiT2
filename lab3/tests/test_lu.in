3 3
5 3 2
1 2 0
3 0 4