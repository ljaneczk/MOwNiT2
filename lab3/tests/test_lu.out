[[ 1.          0.          0.        ]
 [ 0.2         1.          0.        ]
 [ 0.6        -9/7         1.        ]]
[[ 5.          3.          2.        ]
 [ 0.          1.4        -0.4       ]
 [ 0.          0.          16/7      ]]